doc/users/whats_new.rst

```diff
@@ -220,184 +366,60 @@ from :func:`~matplotlib.pyplot.figure`, it can also be used there::
-SVG file to be deterministic. The downside of this setting is that if
-more than one file is generated using deterministic identifiers
-and they end up as parts of one larger document, the identifiers can
-collide and cause the different parts to affect each other.
-
-These features are now enabled in the tests for the PDF and SVG
-backends, so most test output files (but not all of them) are now
-deterministic.
-
+Plotting a :func:`~matplotlib.axes.Axes.quiverkey` now admits the
+``angle`` keyword argument, which sets the angle at which to draw the
+key arrow.
 
 Colormap reversed method
 ------------------------
 
-The methods :meth:`~matplotlib.colors.LinearSegmentedColormap.reversed` and
-:meth:`~matplotlib.colors.ListedColormap.reversed` return a reversed
+The methods :meth:`matplotlib.colors.LinearSegmentedColormap.reversed` and
+:meth:`matplotlib.colors.ListedColormap.reversed` return a reversed
 instance of the Colormap. This implements a way for any Colormap to be
 reversed.
 
 
-`Collection` offsets are no longer implicitly flattened
--------------------------------------------------------
-
-`~matplotlib.collections.Collection` (and thus both 2D
-`~matplotlib.axes.Axes.scatter` and 3D
-`~mpl_toolkits.mplot3d.axes3d.Axes3D.scatter`) no
-longer implicitly flattens its offsets.  As a consequence, ``scatter``'s ``x``
-and ``y`` arguments can no longer be 2+-dimensional arrays.
-
-
-`Artist.setp` (and `pyplot.setp`) accept a `file` argument
-----------------------------------------------------------
+`Artist.setp` (and `pyplot.setp`) accept a ``file`` argument
+------------------------------------------------------------
 
 The argument is keyword-only. It allows an output file other than
 `sys.stdout` to be specified. It works exactly like the ``file`` argument
 to `print`.
 
 
+``streamplot`` streamline generation more configurable
+------------------------------------------------------
 
-Maximum streamline length and integration direction can now be specified
-------------------------------------------------------------------------
-
-This allows to follow the vector field for a longer time and can enhance the
-visibility of the flow pattern in some use cases.
+The starting point, direction, and length of the stream lines can now
+be configured.  This allows to follow the vector field for a longer
+time and can enhance the visibility of the flow pattern in some use
+cases.
 
 
-`Axis.set_tick_params` now responds to 'rotation'
--------------------------------------------------
+`Axis.set_tick_params` now responds to ``rotation``
+---------------------------------------------------
 
 Bulk setting of tick label rotation is now possible via
-:func:`~matplotlib.axis.Axis.set_tick_params` using the ``rotation`` keyword.
+:func:`~matplotlib.axis.Axis.set_tick_params` using the ``rotation``
+keyword.
 
-Example
-~~~~~~~
 ::
 
     ax.xaxis.set_tick_params(which='both', rotation=90)
 
+
```

Comment:
just below "Shading can now be toggled etc" to be consistent with the style of other headings